account_financial_report_webkit/report/common_partner_balance_reports.py

```diff
@@ -259,7 +261,7 @@ def compute_partner_balance_data(self, data, filter_report_type=None):
         # get details for each accounts, total of debit / credit / balance
         accounts_by_ids = self._get_account_details(
             account_ids, target_move, fiscalyear, main_filter, start, stop,
-            initial_balance_mode)
+            initial_balance_mode, context=lang_ctx)
```

Comment:
This is a very minimal change. By only adding context=lang_ctx to those places in the code where there was no context passing before I fix the multi-language problem of the account.account,name field without risk to break something else. 
I tested all reports with this change and everything work fine, hence I prefer that we merge it this way. 
Seen the fact that many methods within the standard account module react differently depending on what is in the context, I think we risk to break a set of reports which work fine for several years now (and we all know how sensitive accountants are when this module would start to return wrong financial data).